.github/workflows/main.yaml

```diff
@@ -0,0 +1,17 @@
+name: main
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.22' ]
+    name: go ${{ matrix.go }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go }}
+      - run: go test -race -v ./...
+        env:
+          GOEXPERIMENT: rangefunc
```

Comment:
At this point, there is no need to enable rangefunc.

Reply:
ah, sorry, this is from the next PR :see_no_evil: 

slipped in by accident → removed, PTAL